docs/integrations/sources/instagram.md

```diff
@@ -70,24 +62,20 @@ The Instagram source connector supports the following [sync modes](https://docs.
 * [Stories](https://developers.facebook.com/docs/instagram-api/reference/ig-user/stories/)
   * [Story Insights](https://developers.facebook.com/docs/instagram-api/reference/ig-media/insights)
 
-For more information, see the [Instagram API](https://developers.facebook.com/docs/instagram-api/) and [Instagram Insights API documentation](https://developers.facebook.com/docs/instagram-api/guides/insights/).
-
-
 ### Features
```

Comment:
@Amruta-Ranade, should I delete this table and add a note under incremental sync modes that says "Incremental sync modes are only available for the User Insights stream"? It seems slightly repetitive.

Reply:
Yep